### DISCUSSION TITLE
Einfacher Abfahrer mit eigenem Kurs?

### DISCUSSION THREAD

**Discussion by BenVape** [Courseplay/Courseplay_FS22#2252] [Q&A] [answered]
Hallo Community und Devs. 
Habe mich jetzt einige Zeit mit CP für LS22 auseinandergesetzt, finde aber keine Möglichkeit einen simplen Abfahrer mit eigenem Kurs fahren zu lassen. 
Gibt es diese Funktion denn gar nicht mehr? 

Hintergrund ist der: 
Ich arbeite gerne mit Motherbins. 
Sprich, ich tanke meine Drescher selber mit Überladewagen im Feld ab, packe alles in den Motherbin und kümmere mich später ums Abfahren. 
Früher hab ich das einfach einen CP-Helfer machen lassen, schnell einen eigenen Kurs eingefahren und den CP-Helfer mit einem kleinen Gespann (Traktor plus Anhänger) das Entleeren vom Motherbin zum Verkauf erledigen lassen während ich mich mit Überlader-Gespann um die Drescher kümmere. 
Scheinbar geht das jetzt aber nicht mehr, und der ach so tolle LS-eigene Helfer bietet da scheinbar auch keine entspr. Option. 
Jedenfalls findet sich der Motherbin nicht als Ladestelle. 
Auch das Entleeren von selbst platzierten Silos auf Mod-Karten verweigert der LS-Helfer, mit der Begründung er wäre blockiert (was nicht der Fall ist). 

Ich verstehe dass CP22 ein anderes Konzept verfolgt als die bisherigen CPs, aber die Funktion eigene Kurse für einfache Abfahrer erstellen zu können ist eigentlich der Kern von CP, und das Fehlen selbiger Funktion würde CP für mich komplett nutzlos machen. 

Ist nicht böse oder abwertend gemeint, das bitte nicht falsch verstehen. 
Aber DIE Kernfunktion von CP einfach dem LS-Helfer (welcher nach wie vor dumm wie Brot ist) überlassen zu wollen halte ich für fragwürdig. 
Oder übersehe ich einfach die entsp. CP-Funktion....?

**Comment by Tensuko** [comment, marked as answer]
> Hallo Community und Devs. Habe mich jetzt einige Zeit mit CP für LS22 auseinandergesetzt, finde aber keine Möglichkeit einen simplen Abfahrer mit eigenem Kurs fahren zu lassen. Gibt es diese Funktion denn gar nicht mehr?

Nein, gibt es nicht.
Dafür gibt es AutoDrive, wie schon so oft kommuniziert.

> und der ach so tolle LS-eigene Helfer bietet da scheinbar auch keine entspr. Option.

Der LS-Helfer kann es Teils auch.

> Ich verstehe dass CP22 ein anderes Konzept verfolgt als die bisherigen CPs, aber die Funktion eigene Kurse für einfache Abfahrer erstellen zu können ist eigentlich der Kern von CP, und das Fehlen selbiger Funktion würde CP für mich komplett nutzlos machen.

Wenn es nur um das Fahren für dich geht bei CP, weil es sonst ja komplett Nutzlos ist, guck dir halt AutoDrive an.
Und nein, der Kern von CP ist NICHT das hin und her fahren, sondern das Arbeiten auf dem Feld.
Kursgenerator, Handling von verschiedensten Arbeitsgeräten, etc...

> Ist nicht böse oder abwertend gemeint, das bitte nicht falsch verstehen. Aber DIE Kernfunktion von CP einfach dem LS-Helfer (welcher nach wie vor dumm wie Brot ist) überlassen zu wollen halte ich für fragwürdig. Oder übersehe ich einfach die entsp. CP-Funktion....?

Wie oben geschrieben, das hin und herfahren ist NICHT die Kernfunktion von CP.
Feldarbeit, dafür ist CP da!

**Comment by rohne83** [comment]
Verstehe nicht was das Problem ist. Einfach MotherBin an den Feldrand stellen, ULW mit CP einstellen, und wenn er voll ist findet der CP-ULW findet die MotherBin selbständig
![fsScreen_2023_01_17_16_43_29](https://user-images.githubusercontent.com/71030138/212943675-68ae95f8-07b7-40e4-9716-3f8967e790ba.png)

Wenns dir um das Befüllen von LKWs an der Mother Bin geht: macht doch eh keinen großen Unterschied ob du den Kurs mit AD oder CP einfährst. Die Straßenfahrt zum Silo bekommt AD aber sowieso deutlich besser hin als CP das konnte, und deswegen braucht CP das Feature im LS22 nicht mehr.

**Reply by Tensuko** [reply]
Naja besser würde ich nicht unbedingt sagen(das automatische verteilen von CP fand ich schon ganz nice), aber so ein Straßennetz hat eben seine Vorteile :)
